app/presenters/policy_area_content_item_presenter.rb

```diff
@@ -0,0 +1,94 @@
+class PolicyAreaContentItemPresenter
+
+  def initialize(policy_area)
+    @policy_area = policy_area
+  end
+
+  def exportable_attributes
+    {
+      "base_path" => base_path,
+      "format" => "policy_area",
+      "content_id" => content_id,
+      "title" => title,
+      "description" => description,
+      "public_updated_at" => public_updated_at,
+      "update_type" => "major",
+      "publishing_app" => "policy-publisher",
+      "rendering_app" => "finder-frontend",
+      "routes" => routes,
+      "details" => details,
+      "links" => {
+        "organisations" => organisations,
+        "topics" => [],
```

Comment:
Let's do the latter for the sake of clarity.  You can link current policies to topics so it might be something we have to support.